main/main.ino

```diff
@@ -39,17 +35,17 @@ void setup()
   setupSD(date);
 }
 
-flightPhase runOnPad(int tick);
-flightPhase runAscending(int tick);
-flightPhase runDescending(int tick);
-flightPhase runPostFlight(int tick);
+flightPhase runOnPad(uint32_t tick);
+flightPhase runAscending(uint32_t tick);
+flightPhase runDescending(uint32_t tick);
+flightPhase runPostFlight(uint32_t tick);
 
 void loop()
 {
-  static int tick = 0;
+  static uint32_t tick = 0;
   static flightPhase status = ONPAD;
-  unsigned int tickTime = getTickTime(status);
-  int x = 1;
+  uint32_t tickTime = getTickTime(status);
+  bool x = 1;
```

Comment:
if we're changing `x` to `bool`, it should be initialized with `true` instead of `1`